index.html

```diff
@@ -212,13 +212,15 @@ <h4>Timothy Kimani</h4>
     </section>
   </main>
   <footer>
-    <h2 class="footer-headline">Official Sponsors</h2>
+    <div class="sponsor_headline">
+      <h2>Sponsors</h2>
+    </div>
     <div class="sponsors">
-      <a href="#"><img src="./assets/airbnb.png" alt="airbnb"></a>
-      <a href="#"><img src="./assets/googlekorea.png" alt="google"></a>
-      <a href="#"><img src="./assets/kakao.png" alt="kakao"></a>
-      <a href="#"><img src="./assets/mozilla.png" alt="mozilla"></a>
-      <a href="#"><img src="./assets/naver.png" alt="naver"></a>
+      <li><a href="#">Google</a></li>
+      <li><a href="#">Mozilla</a></li>
+      <li><a href="#">AirBnB</a></li>
+      <li><a href="#">Kakao</a></li>
+      <li><a href="#">Naver</a></li>
     </div>
     <div class="attribute">
       <p> 2015 Creative Commons Korea Some Rights Reserved </p>
```

Comment:
- [ ] Your home page footer should have a logo and writing. Kindly implement this 👏 
- Your design:
![Annotation 2022-06-24 013235](https://user-images.githubusercontent.com/85113567/175439503-0908cf1d-c7be-4ac2-b360-e28ac7a6ae59.jpg)

- Required design
-  ![Annotation 2022-06-24 013253](https://user-images.githubusercontent.com/85113567/175439524-2ceb5e1f-3131-4a9f-9674-4aa9918f15e7.jpg)